blockchain/My-Wallet-V3-iOS
# fix(ExchangeCreateInteractor): IOS-1435 record last ether tx and prevent sendi…

…ng ether if waiting for confirmation

## Objective

Maintain same ETH send behavior in Exchange as in the regular send screen.

## Description

- Added recording of last ETH tx after sending ETH from exchange
- Show error message to inform user that they must wait for the last ETH tx to confirm before sending again

## How to Test

- Send ETH from regular Send tab or via Exchange
- See error message in Exchange when attempting to send from ETH 

## Screenshot/Design assessment

![simulator screen shot - iphone 6 - 2018-10-11 at 21 37 34](https://user-images.githubusercontent.com/10579422/46842971-a87b9b80-cd9e-11e8-9241-01f847f0b7b6.png)

## Merge Checklist
